api/sentry/utils.py

```diff
@@ -11,18 +11,31 @@
     "firefox-ios": {
         "title": ":testops-apple: iOS",
         "looker_dashboard_url": (
-            "https://mozilla.cloud.looker.com/dashboards/2381"
+            "https://mozilla.cloud.looker.com/dashboards/"
+            "2667?Sentry+Project+ID=6176941&Created+Month=30+days"
         ),
         "confluence_report_url": (
             "https://mozilla-hub.atlassian.net/wiki/spaces/"
             "MTE/pages/1631911951/iOS+Health+Monitor+Report"
         )
     },
     "fenix": {
-        "title": ":testops-android: Android"
+        "title": ":testops-android: Android",
+        "looker_dashboard_url": (
+            "https://mozilla.cloud.looker.com/dashboards/"
+            "2667?Sentry+Project+ID=6375561&Created+Month=30+days"
+        ),
+        "confluence_report_url": (
```

Comment:
This will point to the confluence page but there you would need to add the reports you want to see by creating a section here with either the looks for the graphs or a link to a dashboard...
https://github.com/mozilla-mobile/testops-dashboard/blob/b1865e648f0c41e192307bd144b5075d29a5c23f/config/confluence/yaml/product-health-android.yaml#L11

Reply:
My understanding is that android beta results could be along with release data